integration-test/init_test.go

```diff
@@ -23,6 +25,7 @@ const (
 	initrdName     = "hyper-initrd.img"
 	binaryName     = "runv"
 	rootfsName     = "rootfs"
+	logFileName    = "runv.INFO"
```

Comment:
INFO has already include WARNNING and ERROR, so I skipped them.
Updated to print all the INFO log in log_dir.